themes/classic/templates/catalog/_partials/product-variants.tpl

```diff
@@ -26,7 +26,11 @@
   {foreach from=$groups key=id_attribute_group item=group}
     {if !empty($group.attributes)}
     <div class="clearfix product-variants-item">
-      <span class="control-label">{$group.name}</span>
+      <span class="control-label">{$group.name}{l s=': ' d='Shop.Theme.Catalog'}
```

Comment:
```suggestion
      <span class="control-label">{$group.name}{l s=': ' d='Admin.Global'}
```

@Julievrz what's the reason behind the fact that we would use a backoffice translation domain on a front-office page ? Just to know

Reply:
Hi @Progi1984 sorry it is my mistake I thought it appeared in the BO. @NeOMakinG is right, it should be in Shop.Theme.Catalog...

Reply:
Actually, and I'm sorry for my mistake, but after a double check with Louise it seems that wordings on invoices are exceptions and should therefore be located in Shop.Pdf. 

```suggestion
      <span class="control-label">{$group.name}{l s=': ' d='Shop.Pdf'}
```

Reply:
But it's not invoice at all, it's on the product page

Reply:
Ok I understand. It was not clear for me because the ": " in @Progi1984's PR (21141) is not used the same way as in your PR. In the first case it is in an invoice, in your case it appears on the product page on the FO. So I think you were right from the beginning (Shop.Theme.Catalog). @LouiseBonnard could you confirm please? 😅

Reply:
If it is displayed on the product page in the FO, right, it should be localized in _Shop.Theme.Catalog_.